subprojects/plugins/src/integTest/groovy/org/gradle/java/fixtures/AbstractJavaProjectTestFixturesIntegrationTest.groovy

```diff
@@ -71,10 +71,10 @@ abstract class AbstractJavaTestFixturesIntegrationTest extends AbstractIntegrati
         def skippedJars = getSkippedJars(compileClasspathPackaging)
         def producedJars = [':jar', ':testFixturesJar'] - skippedJars
         executedAndNotSkipped(
-            ":compileJava",
-            ":compileTestFixturesJava",
-            ":compileTestJava",
-            *producedJars
+                ":compileJava",
```

Comment:
Thoughts on reverting these continuation indents back to 4 spaces? I am really not a fan of the 8 space indents and think we should update the intellij config to makes these 4

Reply:
Interesting enough, in our `.editorconfig` file:

```
# Change these settings to your own preference
indent_style = space
indent_size = 4
```

but I have no strong feeling, IntelliJ did that automatically.

Reply:
We would likely need to add a `ij_continuation_indent_size = 4`